chia/wallet/conditions.py

```diff
@@ -482,7 +482,7 @@ def convert_noneness_to_bit(maybe_none: Optional[Any]) -> int:
     @property
     def necessary_args(self) -> list[Program]:
         if self._nothing_committed:
-            raise ValueError("Cannot generate necessary_args for a participant without committment information")
+            raise ValueError("Cannot generate necessary_args for a participant without commitment information")
```

Comment:
This change, while correct, requires changing some test code, as the test code looks for this exact matching string while testing this exception flow